apps/docs/pages/guides/database/column-encryption.mdx

```diff
@@ -0,0 +1,266 @@
+import Layout from '~/layouts/DefaultGuideLayout'
+
+export const meta = {
+  id: 'column-encryption',
+  title: 'Column Encryption',
+  description: 'Use Supabase to store and serve files.',
+  sidebar_label: 'Overview',
+  video: 'https://www.youtube.com/v/J9mTPY8rIXE',
+}
+
+Encrypted Columns for Tables
+
+## Transparent Column Encryption (TCE)
+
+Our recent [blog post](/blog/transparent-column-encryption-with-postgres) describes TCE in-depth. TCE is one of the safest ways to encrypt your data so that it doesn't leak into logs and backups, as well as providing your users with row-level authenticated encryption. TCE is the foundational feature of the Vault, but you can use it on your own tables if you choose to if the Vault isn't sufficient for your needs, for example if you have multiple tables that you wish to have encrypted columns. Any Postgres value that can be cast to `text` or `bytea` can use TCE to encrypt the data that is stored to disk.
```

Comment:
```suggestion
TCE provides a safe way to encrypt your data so that it doesn't leak into logs and backups. It can also provide row-level authenticated encryption.

[Supabase Vault](/docs/guides/database/vault) is built on TCE, but you can use it on your own tables if you choose to if the Vault isn't sufficient for your needs, for example if you have multiple tables that you wish to have encrypted columns. Any Postgres value that can be cast to `text` or `bytea` can use TCE to encrypt the data that is stored to disk.

See the [blog post](/blog/transparent-column-encryption-with-postgres) for an in-depth discussion on TCE.
```